jiwonjang/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUT PORT NUMBER SHOULD BE HERE!
+
+TYPEORM_CONNECTION = mysql
```

```suggestion
TYPEORM_CONNECTION = rdbms
```
다른 변수들에 대해서도 전반적으로 실제 값이 아닌, 중의적인 표현으로 작성될 수 있게 수정해주세요!